src/Features/Core/Portable/Completion/Providers/SymbolCompletionItem.cs

```diff
@@ -66,22 +68,21 @@ internal static class SymbolCompletionItem
                 tags: tags,
                 isComplexTextEdit: isComplexTextEdit);
 
-            item = WithSupportedPlatforms(item, supportedPlatforms);
-            return symbolEncoder(symbols, item);
+            return item;
         }
 
-        public static CompletionItem AddSymbolEncoding(IReadOnlyList<ISymbol> symbols, CompletionItem item)
-            => item.AddProperty("Symbols", EncodeSymbols(symbols));
+        public static ImmutableDictionary<string, string> AddSymbolEncoding(IReadOnlyList<ISymbol> symbols, ImmutableDictionary<string, string> properties)
```

Comment:
Can this be private? (same as `AddSymbolInfo` nelow)

Reply:
Done